gulpfile.js

```diff
@@ -44,6 +44,12 @@ gulp.task('clean:db', cb => rimraf('out/bp/data/storage/core.sqlite', cb))
 // Example: yarn cmd dev:module --public nlu or yarn cmd dev:module --private bank
 gulp.task('dev:module', gulp.series([modules.cleanModuleAssets, modules.createModuleSymlink]))
 
+/**
+ * Example: yarn cmd create:migration --target core --ver 13.0.0 --title "some config update"
+ * target can either be "core" or the name of any module
+ */
+gulp.task('create:migration', core.createMigration)
```

Comment:
we could reverse that ==>  `migration:create` or keeping create: could eventually grow to some other utilitary templating such as create:module, etc